README.md

```diff
@@ -1,21 +1,57 @@
 # AiiDA Prerequisites
 
 This repository adds
-[PostgreSQL](https://www.postgresql.org/) and [RabbitMQ](https://www.rabbitmq.com/) servers on top of [phusion](https://github.com/phusion/baseimage-docker) base image. Additionally, it creates a system profile ready to setup AiiDA under it.
+[PostgreSQL](https://www.postgresql.org/) and [RabbitMQ](https://www.rabbitmq.com/) servers on top of [phusion base image](https://github.com/phusion/baseimage-docker). Additionally, it creates a system profile ready to setup AiiDA under it.
 
 
-## Docker image
+# Docker image
 
-The docker image contains:
- * [PostgreSQL](https://www.postgresql.org/)
- * [RabbitMQ](https://www.rabbitmq.com/)
- * Configured Linux environment for $SYSTEM_USER user
+The docker image:
+ * Is based on [phusion base image](https://github.com/phusion/baseimage-docker).
+ * Installs and launches [PostgreSQL](https://www.postgresql.org/) for $SYSTEM_USER.
+ * Installs and launches [RabbitMQ](https://www.rabbitmq.com/).
+ * Configures Linux environment for $SYSTEM_USER.
+
+## Wait for services
+The image provides a mechanism to wait until the startup script has *launched* all services.
+Example of usage:
+
+```
+$ docker exec --tty $DOCKERID wait-for-services
+```
+This command will exit when all startup scripts are done.
+Note, however, that the mechanism does not check that all services are actually running.
+
+## Docker ARGs
+The the following arguments can be set during *build* time:
+```
+ARG NB_USER="aiida"
+ARG NB_UID="1000"
+ARG NB_GID="1000"
+```
```

Comment:
Perhaps add:
These affect the *default* user name, user id and group id for containers launched from the image.

Reply:
done